lib/ansible/modules/packaging/language/bundler.py

```diff
@@ -135,10 +135,10 @@
 
 
 def get_bundler_executable(module):
+    result = [module.get_bin_path('bundle', True)]
     if module.params.get('executable'):
-        return module.params.get('executable').split(' ')
-    else:
-        return [ module.get_bin_path('bundle', True) ]
+        result = module.params.get('executable').split(' ')
+    return result
```

Comment:
This change isn't good as it makes the module fail if the bundle command isn't found.  need to keep the original if-else structure but can change it to return result.  I also only like to pre-load one answer if the pre-loading is a simple assignment as those are both cheap and do not have side-effects.  In this case, the function being called has a side-effect; in other cases a function being called might have a performance impact. Something like this should satisfy pylint but not suffer these problems:
``` python
def get_bundler_executable(module):
    if module.params.get('executable'):
        result = module.params.get('executable').split(' ')
    else:
        result = [ module.get_bin_path('bundle', True) ]
    return result
```

Reply:
@abadger fix done!